### ISSUE TITLE
Find off-chain list of usernames and add them to indexer

### ISSUE THREAD

**Issue by n1tranquilla** [Granola-Team/mina-indexer#1653]
The source of usernames on minaexplorer.com is not strictly from the chain.

**Comment by jhult** [comment]
https://github.com/garethtdavies/mina-payout/blob/main/foundation_addresses.txt
https://github.com/jrwashburn/mina-addresses/tree/main

**Comment by n1tranquilla** [comment]
Assigning to @robinbb for his opinion

**Comment by robinbb** [comment]
Yes, we should use these, and document their origin (the links above).

**Comment by Isaac-DeFrain** [comment]
Currently in tier-3 https://buildkite.com/granola/mina-indexer-tier-3/builds/916
